lib/apiservers/engine/backends/volume.go

```diff
@@ -53,6 +53,11 @@ var validDriverOptsKeys = map[string]struct{}{
 	DriverArgImageKey:     {},
 }
 
+var supportedVolDrivers = map[string]struct{}{
+	"local":   struct{}{},
```

Comment:
We only support one driver currently. Check with @matthewavery why we have "local" - it may be that it's the default supplied by the client. We could do with a comment in this code to explain what the difference is between local & vsphere.

It's quite possible that we want to map local -> vsphere if specified (assuming my guess about it being the client's default value is correct), but I don't think we should return it in the list. Returning two drivers implies that there's a choice and that they have different behaviours.